apps/web/src/scenes/NftDetailPage/LinkToFullPageNftDetailModal.tsx

```diff
@@ -108,7 +108,7 @@ export default function LinkToFullPageNftDetailModal({
   );
 }
 
-const StyledAnchor = styled.div`
+const StyledAnchor = styled.a`
```

Comment:
not sure why this was a div in the first place - maybe changing it an `a` tag breaks something im not aware of?

Reply:
does it mess with the Next Link ? 🤔